plugins/modules/fusion_info.py

```diff
@@ -907,8 +907,8 @@ def main():
         info["storage_endpoints"] = generate_se_dict(fusion)
     if "api_clients" in subset or "all" in subset:
         info["api_clients"] = generate_api_client_dict(fusion)
-    if "nigs" in subset or "all" in subset:
-        info["network_interface_groups"] = generate_nig_dict(fusion)
+    if "network_interface_groups" in subset or "all" in subset or "nigs" in subset:
+        info["network_interface_groups"] = generate_nigs_dict(fusion)
```

Comment:
Put a `module.warn()` here to state that `nigs` will be deprecated in 1.7.0 if `nigs` is in `subset`

Reply:
You can then deprecate `nigs` after Collection 1.6.0 has been released ready for 1.7.0

Reply:
done